ambry-rest/src/main/java/com.github.ambry.rest/NettyServerFactory.java

```diff
@@ -87,6 +77,14 @@ protected void initChannel(SocketChannel ch) {
    */
   @Override
   public NioServer getNioServer() {
-    return new NettyServer(nettyConfig, nettyMetrics, channelInitializer);
+    return new NettyServer(nettyConfig, nettyMetrics, channelInitializers);
+  }
+
+  /**
+   * Get the {@link ChannelInitializer}s to be used when constructing {@link NettyServer}s.
+   * @return an unmodifiable map that maps port numbers to {@link ChannelInitializer}s.
+   */
+  Map<Integer, ChannelInitializer<SocketChannel>> getChannelInitializers() {
```

Comment:
The only other way of testing that 2 channel initializers are provided would be to start up a server and see if the ports are bound. I could also just remove this test, since a similar test wasn't there before